vaadin-archetype-liferay-portlet/src/main/resources/archetype-resources/src/main/webapp/WEB-INF/liferay-portlet.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<!DOCTYPE liferay-portlet-app PUBLIC "-//Liferay//DTD Portlet Application 7.1.0//EN" "http://www.liferay.com/dtd/liferay-portlet-app_7_1_0.dtd">
+
+<liferay-portlet-app>
+    <portlet>
+        <portlet-name>${portletName}</portlet-name>
+        <instanceable>false</instanceable>
+        <css-class-wrapper>my-portlet-css</css-class-wrapper>
```

Comment:
May we use `portletId` property here (e.g. `${portletId}-css`)? Perhaps also hard-coded value is fine.

Reply:
Yes, let's use portlet name